### PR TITLE
Update some links from bitbucket to GitHub

### PR DESCRIPTION
**Pull request by Banner-Keith**
I could not find equivalent links for all of these. But I put in the ones I did find.

### CODE REVIEW

File: Downloads.md [hunk 1]
```diff
@@ -109,4 +109,4 @@ Don't forget to also check-out the [many packages on NuGet that build on top of
 
   * [Full Package](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.0-bin.zip)
   * [Binaries](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.0-bin.zip)
-  * [Sources](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.0-src.zip)
\ No newline at end of file
+  * [Sources](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.0-src.zip)
```
Comment (atifaziz) [inline Downloads.md:112]: This seems like a whitespace change only. Can we revert this, please?

File: Downloads.md [hunk 2]
```diff
@@ -51,10 +51,10 @@ If you downloaded the _core binary_ then you may need additional driver binaries
 | Database | Driver Binaries | DDL Script | Notes |
 |----------|-----------------|------------|-------|
 | [Microsoft Access](http://office.microsoft.com/access/) | Included in .NET Framework | [MS Access DDL (WSH) Script](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-db-mkmdb.vbs) | The DDL script is also embedded in ELMAH. You may need it if you don't have sufficient rights to create the database at runtime. |
-| [Microsoft SQL Server](http://www.microsoft.com/sqlserver/) | Included in .NET Framework | [MS SQL Server DDL Script](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-db-SQLServer.sql) | - |
-| [MySQL](http://www.mysql.com/) | [MySQL Binaries](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-lib-MySql.zip) | [MySQL DDL Script](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-db-MySql.sql) | - |
+| [Microsoft SQL Server](http://www.microsoft.com/sqlserver/) | Included in .NET Framework | [MS SQL Server DDL Script](https://github.com/elmah/SqlErrorLog/blob/master/src/SQLServer.sql) | - |
```
Comment (atifaziz) [inline Downloads.md:54]:
```suggestion
| [Microsoft SQL Server](http://www.microsoft.com/sqlserver/) | Included in .NET Framework | [MS SQL Server DDL Script](https://github.com/elmah/Elmah/blob/v1.2-sp2/src/Elmah/SQLServer.sql) | - |
```

File: Downloads.md [hunk 3]
```diff
@@ -51,10 +51,10 @@ If you downloaded the _core binary_ then you may need additional driver binaries
...
+| [MySQL](http://www.mysql.com/) | [MySQL Binaries](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-lib-MySql.zip) | [MySQL DDL Script](https://github.com/elmah/MySqlErrorLog/blob/master/src/MySql.sql) | - |
```
Comment (atifaziz) [inline Downloads.md:55]:
```suggestion
| [MySQL](http://www.mysql.com/) | [MySQL Binaries](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-lib-MySql.zip) | [MySQL DDL Script](https://github.com/elmah/Elmah/blob/v1.2-sp2/src/Elmah/MySql.sql) | - |
```

File: Downloads.md [hunk 4]
```diff
@@ -51,10 +51,10 @@ If you downloaded the _core binary_ then you may need additional driver binaries
...
 | [Oracle](http://www.oracle.com/) | Included in .NET Framework 2.0+ | [Oracle DDL Script](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-db-Oracle.sql) | - |
-| [PostgreSQL](http://www.postgresql.org/) | [PostgreSQL Binaries](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-lib-Pgsql.zip) | [PostgreSQL DDL Script](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-db-Pgsql.sql) | - |
+| [PostgreSQL](http://www.postgresql.org/) | [PostgreSQL Binaries](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-lib-Pgsql.zip) | [PostgreSQL DDL Script](https://github.com/elmah/PgsqlErrorLog/blob/master/src/Pgsql.sql) | - |
```
Comment (atifaziz) [inline Downloads.md:57]:
```suggestion
| [PostgreSQL](http://www.postgresql.org/) | [PostgreSQL Binaries](https://bitbucket.org/project-elmah/main/downloads/ELMAH-1.2-lib-Pgsql.zip) | [PostgreSQL DDL Script](https://github.com/elmah/Elmah/blob/v1.2-sp2/src/Elmah/Pgsql.sql) | - |
```

**Review by atifaziz — CHANGES_REQUESTED** [review]
Thank you this PR. I have suggested edits because pointing to `master` is never a good idea since the cheese could get moved and then those links will go 404. I have essentially pinned the links to the [`v1.2-sp2` tag](https://github.com/elmah/Elmah/releases/tag/v1.2-sp2) instead.

**Comment by Banner-Keith** [comment]
@atifaziz I have committed your changes and reverted the new line character at the end of the file.

**Review by atifaziz — APPROVED** [review]
Thank you again for those updates and follow-up!

**Pull request merged**